e2e/suite1/connections-actions.test.ts

```diff
@@ -0,0 +1,295 @@
+import * as connections from '../helpers/connection-actions';
+import disableTransitions from '../helpers/disableTransitions';
+import insertFixtures from '../helpers/insertFixtures';
+import { adminLogin, logout } from '../helpers/login';
+import proxyMock from '../helpers/proxyMock';
+
+describe('connections', () => {
+  beforeAll(async () => {
+    await insertFixtures();
+    await proxyMock();
+    await adminLogin();
+    await disableTransitions();
+  });
+
+  afterAll(async () => {
+    await logout();
+  });
+
+  it('should navigate to an entities connections view', async () => {
+    await expect(page).toClick(
+      '#filtersForm > div:nth-child(2) > ul > li > ul > li:nth-child(6) > label > span.multiselectItem-name > span'
+    );
+    await expect(page).toClick('.item-name', {
+      text: 'Acevedo Buendia et al (Discharged and Retired Employees of the Office of the Comptroller)',
+    });
+    await expect(page).toClick(
+      '#app > div.content > div > div > div > aside.side-panel.metadata-sidepanel.is-active > div.sidepanel-footer > span > a'
+    );
+    await expect(page).toMatchElement('.item-name', {
+      text: 'Acevedo Buendia et al (Discharged and Retired Employees of the Office of the Comptroller)',
+    });
+    await expect(page).toClick('#tab-connections');
+  });
+
+  it('should remove every hub except the first one', async () => {
+    await connections.clickEdit();
+    await connections.removeHub(2);
+    await connections.removeHub(3);
+    await connections.removeHub(4);
+    await connections.removeHub(5);
+    await connections.clickSave();
+  });
+
+  it('should add new hub with relation group "País"', async () => {
+    await connections.reloadPage();
+    await connections.clickEdit();
+    await connections.addHub('Court');
+    await connections.addNewRelTypeToHub(2, 'País');
+  });
+
+  it('should add some countries', async () => {
+    await connections.addNewConnection(2, 1, 'Bolivia');
+    await connections.addNewConnection(2, 1, 'Brazil');
+    await connections.addNewConnection(2, 1, 'Tracy Robinson');
+    await connections.addNewConnection(2, 1, 'Chile');
+    await connections.addNewConnection(2, 1, 'Colombia');
+  });
+
+  it('should add new relation group "Firmantes"', async () => {
+    await connections.addNewRelTypeToHub(2, 'Firmantes');
+  });
+
+  it('should add some people to "Firmantes"', async () => {
+    await connections.addNewConnection(2, 2, 'Máximo Cisneros Sánchez');
+    await connections.addNewConnection(2, 2, 'Margarette May Macaulay');
+  });
+
+  it('should save the relations', async () => {
+    await connections.clickSave();
+  });
+
+  it('should move the person between to countries to the correct group', async () => {
```

Comment:
Please fix this description.  Is it `two countries?`, is it `from countries to the correct group`? Thanks.

Reply:
Fixed.